src/PageManagerServiceProvider.php

```diff
@@ -41,6 +41,14 @@ public function boot()
         $this->publishes([__DIR__.'/resources/lang' => resource_path('lang/vendor/backpack')], 'lang');
 
         $this->mergeConfigFrom(__DIR__.'/config/pagemanager.php', 'backpack.pagemanager');
+        // LOAD THE VIEWS
+        // - first the published/overwritten views (in case they have any changes)
+        $customViewsFolder = resource_path('views/vendor/backpack/crud');
```

Comment:
@pleone you meant ```views/vendor/backpack/pagemanager``` here, right?

Reply:
@tabacitu well not really. 
Actually the only view published is a field ( select_page_template.blade.php ) and at the moment it is published in the crud folder. I mean, i think it's ok that a filed goes in that folder with the other fields.  
There are no other views in this package, so i thought to leave it this way.
What do you think?

Reply:
Sorry for the late reply @pleone . Oh you're right - the only view is the custom field.

**In this case, I don't think it really makes sense to merge this PR, and load the custom views in PageManagerServiceProvider.** It would create the new ```pagemanager::``` namespace for PageManager views, yes - but _we don't have any pagemanager-specific views_, as you said. If the only view is a custom CRUD field, that's already accessible using ```crud::fields.select_page_template```.

Maybe I don't see a use case you're trying to achieve. What would this PR help you do, that you couldn't previously do?

Thanks, cheers!

Reply:
Heads-up @pleone , just went through this again and I agree with my previous self 😂 so I'm going to close this. Feel free to reply if you think I was wrong on this one - I regularly am 😂

Cheers! Really appreciate the time you took to submit the PR though. 🙏